packages/react/src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss

```
@@ -16,8 +16,8 @@
 
   /* the spinner was a little too big and causing the row to scroll so need to scale down a bit */
   .#{$prefix}--loading--small {
-    width: 1.875rem;
-    height: 1.875rem;
+    width: 1rem;
+    height: 1rem;
```

```suggestion
    width: $spacing-05;
    height: $spacing-05;
```

Thanks, @Shady-Nessim. Let's use the spacing token here instead.